### ISSUE TITLE
https://nx.dev/latest/angular/getting-started/console

### ISSUE THREAD

**Issue by abergquist** [nrwl/nx#5243]
Hi,

I see a couple of grammatical issues at https://nx.dev/latest/angular/getting-started/console:

1. "Pallete" needs to be replaced with "Palette" in both occurrences of the following text:

> From the Command Pallete

2.  "satified" needs to be replaced with "satisfied" in the following sentence:

> When you're satified with the constructed command, choose the Execute command at the top of the list.

TIA,

Art Bergquist

**Comment by mandarini** [comment]
Hi @abergquist , thank you for this! Would you like to give this a shot? I mean, would you like to try and make the changes yourself, as a contribution? I can help you through if you want! :)

**Comment by abergquist** [comment]
This is a strange one; I just cloned the repo and couldn't find the typos.  When I then look at docs\shared\console.md, it appears that the typos have been corrected in that file.  When I go back to https://nx.dev/latest/angular/getting-started/console, though, I still see the errors (even after firing up Developer Tools and hard-refreshing).  BTW, I now see 2 instances of "satified" that need to be replaced with "satisfied".
